meta.yaml

```
@@ -29,6 +29,7 @@ requirements:
     - python >=3.7
     - pyyaml >=5.4.1
     - ruamel.yaml >=0.17.10
+    - jsonschema >=4.7.2
```

Can we have a max version?
```suggestion
    - jsonschema >=4.7.2, <4.8
```